src/app/[slug]/page.tsx

```diff
@@ -52,7 +52,7 @@ const Page = async ({ params, searchParams }: Props) => {
           headline={featuredDrop}
           staticHeadline={!!dropAddress}
         />
-        <section className="w-full font-text p-1">
+        <section className="w-full font-text p-1 bg-ocs-light-gray shadow-large rounded-3xl">
```

Comment:
I think our top padding is off in this section relative to the home page
![image](https://github.com/base-org/onchainsummer.xyz/assets/41811693/d52002d7-27af-40a2-b48e-53095f127515)

Reply:
Yes! Thanks for catching that.